app/boards/shields/cradio/Kconfig.shield

```diff
@@ -0,0 +1,5 @@
+# Copyright (c) 2020 Pete Johanson
```

Comment:
Ditto.

```suggestion
# Copyright (c) 2020 The ZMK Contributors
```

Reply:
added# Copyright (c) 2020 The ZMK Contributors
# SPDX-License-Identifier: MIT